contracts/dao/ListaRevenueDistributor.sol

```diff
@@ -112,6 +124,58 @@ contract ListaRevenueDistributor is Initializable, AccessControlUpgradeable {
         emit RevenueDistributed(token, amount0, amount1);
     }
 
+    /**
+     * @dev distribute tokens to autoBuybackAddress and revenueWalletAddress
+     *      according to distributeRate excluding cost
+     *
+     * @param tokens, token to distribute
+     * @param costs, amount directly to cost address
+     */
+    function distributeTokensWithCost(address[] memory tokens, uint256[] memory costs)
+        external
+        onlyRole(MANAGER)
+    {
```

Comment:
Here a check can be added to verify that the length of tokens and costs are same.